gateway/api/views.py

```diff
@@ -1,4 +1,4 @@
-"""
+"""OA
```

Comment:
What it means `OA` here @akihikokuroda , just for curiosity.

Reply:
Thanks! It's nothing.  I put it by mistake. I take it out.